library/Reporting/Web/Forms/TemplateForm.php

```diff
@@ -247,6 +247,14 @@ protected function addColumnSettings($name, $label)
             ]
         ]);
 
+        $valueType = $this->getValue($type, 'none');
+        $populated = $this->getPopulatedValue($value);
+        if ($valueType === 'image' && ! $populated instanceof UploadedFile) {
+            $this->clearPopulatedValue($value);
+        } elseif ($valueType !== 'image' && $populated instanceof UploadedFile) {
+            $this->clearPopulatedValue($value);
+        }
```

Comment:
Please combine this into a single if condition.

Reply:
How does that make any difference?

Reply:
The code in `If` and `elseif` is identical, so why not combine the two?